packages/logs/src/domain/assembly.ts

```diff
@@ -52,6 +54,10 @@ export function startLogsAssembly(
         messageContext
       )
 
+      if (!isEmptyObject(commonContext.user)) {
+        log.usr = shallowClone(commonContext.user)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏what is your reasoning for using a shallow clone here?